UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Services/FssService.cs

```diff
@@ -405,16 +466,16 @@ private CreateBatchRequestModel CreateBatchRequestModel(Batch batchType)
                 };
             }
             //Temporary Upload Code Start
-            else if (batchType == Batch.BesBaseZipBatch || batchType == Batch.BesUpdateZipBatch)
+            else if (batchType is Batch.BesBaseZipBatch or Batch.BesUpdateZipBatch or Batch.BesChangeZipBatch)
             {
-                createBatchRequest = new()
+                createBatchRequest = new CreateBatchRequestModel
                 {
                     BusinessUnit = _fssApiConfiguration.Value.BessBusinessUnit,
                     ExpiryDate = DateTime.UtcNow.AddDays(28).ToString("yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture),
-                    Acl = new Acl()
+                    Acl = new Acl
                     {
-                        ReadUsers = string.IsNullOrEmpty(_fssApiConfiguration.Value.BessReadUsers) ? new() : _fssApiConfiguration.Value.BessReadUsers.Split(",").ToList(),
-                        ReadGroups = string.IsNullOrEmpty(_fssApiConfiguration.Value.BessReadGroups) ? new() : _fssApiConfiguration.Value.BessReadGroups.Split(",").ToList(),
+                        ReadUsers = string.IsNullOrEmpty(_fssApiConfiguration.Value.BessReadUsers) ? new List<string>() : _fssApiConfiguration.Value.BessReadUsers.Split(",").ToList(),
```

Comment:
Same as above with the use of 'ToList()'

Reply:
Resolved.